base_tier_validation_correction/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+Person that can use the tier review correction, must has access right to group **Tier Review Correction**
```

Comment:
`**Tier Review Correction**` is bold, I think you wanted to use italics? then it is `*Tier Review Correction*`